bugbot/rules/perfalert_resolved_regression.py

```diff
@@ -10,10 +10,26 @@
 from bugbot.bzcleaner import BzCleaner
 from bugbot.constants import BOT_MAIN_ACCOUNT
 
+RESOLUTION_KEYWORDS = (
+    "backedout",
+    "backed out",
+    "back out",
+    "backout",
+    "wontfix",
+    "invalid",
+    "incomplete",
+    "duplicate",
+    "fixed",
+    "resolved",
+    "resolve",
+    "resolution",
+)
+
 
 class PerfAlertResolvedRegression(BzCleaner):
-    def __init__(self, *args, **kwargs):
-        super().__init__(*args, **kwargs)
+    def __init__(self, acceptance_time=86400):
```

Comment:
Adding docstrings would be beneficial, and I would opt for a more descriptive name, such as `max_hours_before_status`.

Reply:
I used seconds here instead of hours.